aopy/analysis/base.py

```diff
@@ -366,10 +366,34 @@ def calc_freq_domain_amplitude(data, samplerate, rms=False):
     freq_data = np.fft.fft(data, axis=0)
     length = np.shape(freq_data)[0]
     freq = np.fft.fftfreq(length, d=1./samplerate)
-    data_ampl = abs(freq_data[freq>=0,:])*2/length # compute the one-sided amplitude
+
+    # Only take non-negative frequency components
     non_negative_freq = freq[freq>=0]
```

Comment:
do you need this variable?

Reply:
Which variable? `non_negative_freq`?

Reply:
yeah non_negative_freq

Reply:
Yeah it's useful to have the frequencies (hz) corresponding to the freq domain values. Another aopy function calls this one and seems like `non_negative_freq` is used there as the x-axis in plotting spectrograms. One of my functions uses `non_negative_freq` to find the indices of the experimental freqs, since we only need the transfer functions at those frequencies.

Reply:
Yeah it's useful to have the frequencies (hz) corresponding to the freq domain values. Another aopy function calls this one and seems like `non_negative_freq` is used there as the x-axis in plotting spectrograms. One of my functions uses `non_negative_freq` to find the indices of the experimental freqs, since we only need the transfer functions at those frequencies.